Add note about GitHub callback URL in docs (#517)
the-grey-group/datalab: pydatalab/docs/config.md

For GitHub, you must register a [GitHub OAuth
application](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app) for your instance, providing the client ID and secret in the `.env` for the API.

The authorization callback URL in the GitHub app settings should be set to `<YOUR_API_URL>/login/github/authorized`.

> The first login may direct you to this page rather than the UI, depending on your browser.

Reviewer: ```suggestion
A user's first login may direct you to this page rather than the web app, depending on your browser. The user will in that case simply have to navigate back to the URL of the web app and should be logged in.
```

Reviewer: oh oops thats not quite grammatical:

A user's first login may direct them to this page rather than the web app, depending on their browser. The user will in that case simply have to navigate back to the URL of the web app and should be logged in.

> A user's first login may direct you to this page rather than the web app, depending on your browser. The user will in that case simply have to navigate back to the URL of the web app and should be logged in.

Author: ```suggestion
A user's first login may direct them to this page rather than the web app, depending on your browser. 
The user will then simply have to navigate back to the URL of the web app, where they should find themselves to be logged in.
```

For GitHub, you must register a [GitHub OAuth
application](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app) for your instance, providing the client ID and secret in the `.env` for the API.

The authorization callback URL in the GitHub app settings should be set to `<YOUR_API_URL>/login/github/authorized`.

> A user's first login may direct them to this page rather than the web app, depending on your browser.

Reviewer: ```suggestion
A user's first login may direct them to this page rather than the web app, depending on their browser.
```

I'm sorry 😭
